docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2807,6 +2807,284 @@ have not been included. These are:
+- Someone who is likely to use a fake name might well do so across multiple observers. This would likely increase PRL challenges but will not be included here. 
+
+**Nicknames**
+
+Many people choose to use nicknames instead of their "real" names. A common example is an 
+Alexander who chooses to go by Alex. These individuals might write their nicknames on forms 
+which should be recorded. Here is a list of 1080 names and their `relevant nicknames <https://github.com/carltonnorthern/nicknames/blob/master/names.csv>`_. 
+
+Only those simulants with names in the csv above are eligible to recieve a nickname. First, 
+determine who is eligible for a nickname. Then select simulants for noise. Lastly, replace their 
+name with any of the nicknames included in the csv. If there are multiple options, 
+select at random. 
+
+.. note::
+
+  If the process of determining elgibility prior to selecting simulants for noise is challenging, we can work on finding a simpler approach. 
+
+
+**Miswriting** 
+
+For all numeric miswriting, first select the sample to have noise added. 
+Once the sample is selected, change 1 digit of the number at random to any 
+other random number. 
+
+For example: if the correct SSN is 123456789, a miswriting might look like 123416789. 
+Please confirm that the new digit is an actual error. 
+
+Limitations: 
+
+- This might lead to illogical data, especially for age and dates (e.g., a person who's birthday is 12/87/2000). It is more likely that someone lists an incorrect but still possible birthday/age. However, since the main goal is noise for PRL, we think this is still acceptable. 
+
+**Copy from within Household** 
+
+To allow for confusion between household members, noise will be included to copy data 
+for another person in the household. 
+
+To do this, first determine who is eligible for copying. This is essentially simulants 
+with at least one other person in their household. Note that GQ is not eligible for this 
+type of noise. 
+
+From the eligble simulants, select the sample to have noise added. For those individuals, 
+copy the relevant piece of data from another person in the household. 
+
+Limitations: 
+
+- This oversimplifies some swapping of ages or birthdays between family members. However, it allows up better control over the percent of simulants to receive incorrect information and will likely pose a similar PRL challenege. 
+
+**Month and Day Swap**
+
+Month and day swaps apply to dates. For this, select the sample to have noise added. 
+For those selected, swap the month and day to be in the incorrect position (e.g., December 8, 2022 
+would be listed in MM/DD/YYYY format as 08/12/2022). 
+
+**Incorrect Select** 
+
+Incorrect select applies to a range of data types. For this, select the sample to 
+have noise added. For those selected, randomly select a different option from the 
+others available. Note that for relationship to head of household, this includes 
```

Comment:
I predict that there may be engineering questions about what "others available" means. Is it the unique values present in the observer output? Do we plan to have set lists of the options available for each column?

Reply:
Fair, I tried to clarify this a bit. In my head it is any other option that was available in the original observer. If this is still not clear enough and I should define it explicitly I can